composeApp/src/iosMain/kotlin/common/Locale.ios.kt

```diff
@@ -17,15 +17,16 @@
 package common
 
 import platform.Foundation.NSLocale
-import platform.Foundation.currentLocale
-import platform.Foundation.languageCode
+import platform.Foundation.preferredLanguages
 
 class IOSLocale : Locale {
     override val language: Language
-        get() = NSLocale.currentLocale.languageCode
+        get() = NSLocale
+            .preferredLanguages()
+            .first()
```

Comment:
@easyhooon Now if your language is set to Korean, it will display in Korean regardless of your location.